src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx

```diff
@@ -18,15 +20,18 @@ const AnalysisSpinner = (): React.ReactElement => (
 const BaseAnalysis = (): JSX.Element => {
   const dispatch = useDispatch();
   const [vegaSpec, setVegaSpec] = useState(null);
-  const [renderEditButton, setRenderEditButton] = useState(true);
+  const [renderEditButton, setRenderEditButton] = useState(false);
```

Comment:
@vaidotasp looks like this is referenced in `setActiveButton()` but `setActiveButton()` isn't being used. What are your thoughts on nixing this and `setActiveButton()`?